src/components/form/bounty/index.tsx

```diff
@@ -3,6 +3,8 @@ import { Formik } from 'formik';
 import { observer } from 'mobx-react-lite';
 import React, { useCallback, useEffect, useRef, useState } from 'react';
 import { useIsMobile } from 'hooks';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
```

Comment:
this line for css is what is causing jest to fail it doesn't like importing CSS files like such

Reply:
it should work now there was syntax err in jest config